### PR TITLE
Use COEF value.

### PR DESCRIPTION
**Pull request by p0p4k**
reference : https://github.com/facebookarchive/loop/blob/112975599b1838a33f139a6c6df0fcd9953ee33f/model.py#L117
 `alpha_t = self.COEF * torch.sum(phi_t, 1)`

### CODE REVIEW

**Comment by p0p4k** [comment]
Ok, maybe i am slightly confused now.
https://arxiv.org/pdf/1906.01083.pdf [MELNET] <-- graves attention is implemented according to this paper in our repo as stated in this discussion where tacotron2 was first released in mozillatts. (https://github.com/mozilla/TTS/issues/346).
However, this paper [VoiceLoop] (https://arxiv.org/pdf/1707.06588.pdf) , Page 5, equation 3 has slightly different equation. 
I am trying to match the sigmoid with exp, but it doesn't work. The sigma under square root is missing in our implementation, and so is the 1/sqrt(2*pi) [which is the COEF].
